### PR TITLE
Add initial German translation

### PR DESCRIPTION
**Pull request by vmx**
@rixx if you tell me how to properly add a the formal German translation file, I can also add that.

@astroidex Can you please have a look at the translation (or ping someone else to have a look)?

### CODE REVIEW

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -19,15 +19,15 @@ msgstr ""
...
 
 #: pretalx_public_voting/__init__.py:12
 msgid "A public voting plugin for pretalx"
-msgstr ""
+msgstr "Ein Plugin für eine öffentliche Abstimmung in pretalx"
 
 #: pretalx_public_voting/exporters.py:16
 msgid "Public Voting CSV"
-msgstr ""
+msgstr "CSV der öffentliche Abstimmung"
```
Comment (astroidex) [inline pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:30]: der öffentlichen - ein e fehlt

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -45,161 +45,195 @@ msgid ""
 "\n"
 "The {event.name} organisers\n"
```
Comment (astroidex) [inline pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:46]: "Das {event.name} Organisationsteam"

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -45,161 +45,195 @@ msgid ""
...
 msgstr ""
+"Hi,\n"
+"\n"
+"du hast dich registriert um über Einreichungen von {event.name} abzustimmen\n"
```
Comment (astroidex) [inline pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:50]: "du hast dich registriert, um über Einreichungen zur {event.name} abzustimmen.\n"

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -45,161 +45,195 @@ msgid ""
...
+"Bitte bestätige mit folgendem Link, dass deine E-Mail-Adresse korrekt ist:\n"
+"\n"
+"    {vote_url}\n"
+"\n"
+"Falls du nicht dich nicht für die Stimmabgabe registriert hast, ignorier "
```
Comment (astroidex) [inline pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:55]: "Falls du dich nicht für die Stimmabgabe registriert hast, kannst du  "

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -45,161 +45,195 @@ msgid ""
...
+"diese E-Mail einfach.\n"
```
Comment (astroidex) [inline pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:56]: "diese E-Mail einfach ignorieren.\n"

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -45,161 +45,195 @@ msgid ""
...
+"\n"
+"Vielen Dank, dass du bei der Abstimmung teilnimmst!\n"
```
Comment (astroidex) [inline pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:58]: "Vielen Dank, dass du an der Abstimmung teilnimmst!\n"

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -45,161 +45,195 @@ msgid ""
...
-msgstr ""
+msgstr "Beginn"
 
 #: pretalx_public_voting/forms.py:116
 msgid ""
 "No public votes will be possible after this time. Submissions will not be "
 "publicly visible."
 msgstr ""
+"Nach diesem Zeitpunkt ist keine öffentliche Abstimmung mehr möglich. Die "
+"Einreichungen sind dann icht mehr öffentlich einsehbar."
```
Comment (astroidex) [inline pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:99]: Einreichungen sind dann nicht mehr öffentlich einsehbar."

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -45,161 +45,195 @@ msgid ""
...
 
 #: pretalx_public_voting/forms.py:123
 msgid "Anonymise content"
-msgstr ""
+msgstr "Text anonymisieren"
 
 #: pretalx_public_voting/forms.py:124
 msgid "Hide speaker names and use anonymized content where available?"
-msgstr ""
+msgstr "Wenn möglich, Namen der Vortrageden und anonymisierten Text verwenden?"
```
Comment (astroidex) [inline pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:111]: Vortragenden

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -45,161 +45,195 @@ msgid ""
...
 
 #: pretalx_public_voting/forms.py:133
 msgid "Minimum score"
-msgstr ""
+msgstr "Mindestwertung"
 
 #: pretalx_public_voting/forms.py:134
 msgid "The minimum score voters can assign"
-msgstr ""
+msgstr "Die geringste Wertung. die vergeben werden kann."
```
Comment (astroidex) [inline pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:127]: , statt .

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -45,161 +45,195 @@ msgid ""
...
 
 #: pretalx_public_voting/templates/pretalx_public_voting/settings.html:17
 msgid ""
 "Public voting will show your submissions publicly, and will allow anybody "
 "who provides a valid email address to vote. The email addresses are not "
 "saved, so the process is anonymous."
 msgstr ""
+"Die öffentliche Abstimmung zeigt alle Einreichungen öffentlich an und "
+"ermöglicht es jedem, der eine gültige E-Mail-Adresse hat, abzustimmen. Die "
+"E-Mail-Adressen werden nicht gespeichert, es ist also anonym."
```
Comment (astroidex) [inline pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:175]: "E-Mail-Adressen werden nicht gespeichert. Die Abstimmung ist also anonym."

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -45,161 +45,195 @@ msgid ""
...
 "This page is invalid. Please double-check that you have followed a complete "
 "link to this place."
 msgstr ""
+"Diese Seite ist nicht zulässig. Bitte vergewissere dich, dass du den "
+"gesamten Link verwendet hast."
 
 #: pretalx_public_voting/templates/pretalx_public_voting/submission_list.html:59
 msgid "Click here to sign up for voting."
-msgstr ""
+msgstr "Bitte klicke hier um dich für das Bewerten anzumelden."
```
Comment (astroidex) [inline pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:220]: , vor um

**Review by astroidex — CHANGES_REQUESTED** [review]

**Comment by rixx** [comment]
@vmx ``makemessages`` should create a ``de_Formal`` directory for you, or you could just copy the ``de_DE`` one. You can compare with pretalx core to see what it should look like.

Please let me know when you want this PR merged.

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po (hunk 3 above)
Comment (vmx) [reply pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:50]: Wie wäre es mit "du hast dich bei {event.name} registriert um über die Einreichungen abzustimmen"? Das ist jetzt keine grammatikalische Hochleistung, allerdings funktioniert es halbwegs sowohl mit "FOSSGIS" als auch "Starkbieranstich".

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po (hunk 5 above)
Comment (vmx) [reply pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:56]: Die Idee kam von https://translate.pretalx.com/translate/pretalx/pretalx/de/?checksum=8c269ce491eeedc3, aber ich finde deine Übersetzung besser (werde das also so machen), vielleicht sollte das an der anderen Stelle auch weniger flapsig formuliert werden.

**Comment by vmx** [comment]
@astroidex I've updated the translation.

I took your review (thanks!) and took all the changes, except for one where I used what I propose [in this comment](https://github.com/pretalx/pretalx-public-voting/pull/17#discussion_r571687605). The full changes can be found here: https://github.com/pretalx/pretalx-public-voting/pull/17/commits/e152167a43d31123e2a75ec7e722134b58da38e7

I also found more typos/bugs. Those changes can be found here: https://github.com/pretalx/pretalx-public-voting/pull/17/commits/248b2078a86cea1217b21340840d863788102f8b

And finally I've added a formal translation (is that one used for the FOSSGIS?). In order to make the review easier, here the diff which only shows the difference between the formal and non-formal version: https://github.com/pretalx/pretalx-public-voting/pull/17/commits/502713951f9ab2db2ce0b3daead84fb10d3c3969

File: pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po (hunk 2 above)
Comment (vmx) [reply pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:46]: Accendental comment, this one is the english one.

**Comment by astroidex** [comment]
fixed some typos. I like the formal version for FOSSGIS

**Comment by vmx** [comment]
@rixx This is ready to be merged. If it needs a rebase, let me know and I'll do that.

When this one is merged, the current state is good enough for the FOSSGIS. Could you please then deploy this version?

**Comment by rixx** [comment]
> When this one is merged, the current state is good enough for the FOSSGIS. Could you please then deploy this version?

I've been continually deploying after every PR merge ;)

**Review by rixx — APPROVED** [review]

**Comment by vmx** [comment]
Looks like we broke something. I get a 500 on signup :-/ (i can have a look later tonight if it also happens locally)

**Comment by rixx** [comment]
No, that was me trying to fix the misformatted email. Fixed now.

**Pull request merged**